docs/guide/en/accordion.md

```diff
@@ -0,0 +1,167 @@
+# Accordion
+
+The Accordion widget renders a Bootstrap 5 accordion component.
+
+You can use it to create collapsible content sections where clicking a header toggles the visibility of its associated
+body. 
+
+## Key Features
+- Supports multiple collapsible items with headers and bodies.
+- Configurable to allow multiple items to remain open simultaneously (`alwaysOpen`).
+- Customizable header and toggler tags, attributes, and styles.
+- Optional flush styling to remove borders and rounded corners.
+- Flexible HTML attribute and CSS class management for container, header, body, and toggler elements.
+
+## Quick Start
+To get started, instantiate the `Accordion` widget and add items to it. Here is a basic example:
+
+```php
+<?php
+
+declare(strict_types=1);
+
+use Yiisoft\Bootstrap5\Accordion;
+use Yiisoft\Bootstrap5\AccordionItem;
+?>
+
+// @link https://getbootstrap.com/docs/5.3/components/accordion/#example
+<?= Accordion::widget()
```

Comment:
```suggestion
<?= Accordion::widget()
```

Link to it at the very start of the guide page.

Reply:
done.